src/multi.c

```diff
@@ -152,6 +152,7 @@ void execCommand(client *c) {
     int j;
     robj **orig_argv;
     int orig_argc, orig_argv_len;
+    mstime_t latency;
```

Comment:
For exec, I wonder if we should just allow the exec command to be in the slowlog.

Reply:
This is also an option, but with the current implementation of the `EXEC` command, it is used to indicate the start of a transaction. If the execution time of other commands within the transaction is all accounted for in the `EXEC` command, could this cause ambiguity?

Reply:
I know more people check slowlog then check the latency metrics. Someone also mentioned to me once it was weird that scripts are included even though also have slow internal commands. I don't have a strong opinion though.